migrate/v0_15/migrate_test.go

```diff
@@ -41,7 +41,7 @@ func TestCommittee(t *testing.T) {
 	err = newGenState.Validate()
 	require.NoError(t, err)
 
-	require.Equal(t, len(oldGenState.Committees), len(newGenState.Committees))
+	require.Equal(t, len(oldGenState.Committees), len(newGenState.Committees)-1) // New gen state has 1 more committee
```

Comment:
At the end of the test, export the resulting state 

```
v15Cdc := app.MakeCodec()
	ioutil.WriteFile(filepath.Join("testdata", "kava-8-committee-state.json"), v15Cdc.MustMarshalJSON(newGenState), 0644)
```

There's multiple errors in the way the committees are exported

1. Each committee has a top-level `Type` field, as well as `BaseCommittee.Type`. Is this correct? Why the redundant field?
2. The hard committee top-level type field is not populated.
3. Hard committee is missing quorum and tally_denom

Reply:
1. Fixed in https://github.com/Kava-Labs/kava/pull/941/commits/d930aef7c30fefe8234d9bdb6f359ca60b01bdc1 by removing `BaseCommittee`'s Type field and updating the `GetType` methods. Now there's only one top-level `Type` field.
2. Fixed in https://github.com/Kava-Labs/kava/commit/7ef2087dde9d915a1b79ddcbf6d95c1d4802e041
3. Fixed in https://github.com/Kava-Labs/kava/commit/7ef2087dde9d915a1b79ddcbf6d95c1d4802e041